base_app.htm

```diff
@@ -0,0 +1,60 @@
+<%inherit file="homepage/templates/base.htm" />
+<%! from catalog import models as cmod %>
+
+  <%block name="navitems">
```

Comment:
I would fix the indentation on this block. Tab everything inside of it once, and then unindent that block tab

Reply:
Good idea, that would clean up my code